### DISCUSSION TITLE
Saelor Dev Team Responsiveness

### DISCUSSION THREAD

**Discussion by SpencerPatterson** [saleor/saleor#8507] [General]
I have submitted several requests for custom features over the past week. Looking to spend upwards of $50,000. Haven't heard back from anyone... Does the team normally respond to emails and the filled out form or do I need to find another solution for my headless e-commerce? @maarcingebala

**Comment by karolkielecki** [comment]
@SpencerPatterson Hi Spencer, 

My apologies for not getting back to you sooner. Our sales team is currently attending Web Summit. We'll answer your requests ASAP. Thank you for understanding.
